prototype2/gdgem/GdGemBaseTestData.h

```diff
@@ -66,8 +66,54 @@ std::string vmm3json = R"(
   "geometry_x" : 256,
   "geometry_y" : 256,
 
-  "dump_csv" : false,
-  "dump_h5" : false,
+  "dump_csv" : true,
+  "dump_h5" : true,
   "dump_directory" : ""
 }
 )";
+
+// Truncated data from packet 44 in high_datarates.pcapng
+std::vector<uint8_t> pkt44 = {
```

Comment:
I sort of don't like raw data in headers. This will mess with code line and coverage statistics, and just feels plain hacky. Is there any way to have a truncated pcap file?

Reply:
True it affects code lines (in header files), but not code coverage.
Anyway it is easy to exclude if we do the naming systematically. I name test data somthingTestData.h and I believe the cloc too has an 'exclude' functionality.
Having binary data is also possible but requires more work if it is supposed to work both when working from the build directory as during development, and from Jenkins.

Reply:
There is a way of putting the data in a separate binary file and making that file easily accessible to your testing code:
1. First, find the path to the file in CMake: `get_filename_component(SOME_PATH "test/some_file.txt" DIRECTORY)`.
2. Then pass the path to the compiler: `target_compile_definitions(MyTest
    PRIVATE SOME_PATH="${CMAKE_CURRENT_SOURCE_DIR}/${SOME_PATH}/")`
3. Finally, load the file the usual way using the path name now available through `SOME_PATH`.